package.json

```diff
@@ -88,13 +88,14 @@
     "command-line-usage": "^5.0.5",
     "epipebomb": "^1.0.0",
     "fs-extra": "^7.0.1",
-    "highlight.js": "github:marcoscaceres/highlight.js#7b144b3b83ce296d5b29c5e4fc2ea2a65e9d3f32",
+    "highlight.js": "git+ssh://git@github.com/highlightjs/highlight.js.git#cli",
```

Comment:
Why SSH instead of previous `github:`?

Reply:
No reason, just copy/pasted from GitHub... will switch, as the GitHub: urls are nicer to look at :)